ios/RNShare.m

```diff
@@ -47,6 +47,12 @@
 #import "FacebookStories.h"
 #import "GooglePlusShare.h"
 #import "EmailShare.h"
+
+
+#import "InstagramStories.h"
+// add snapchatshare
```

Comment:
strange whitespace here and the comment seems unnecessary - all the imports should just go one after the other I think?